### PR TITLE
Adding comment summary

### PR DESCRIPTION
**Pull request by adamdavies1**
Adding comment summary for sub criteria
image

### CODE REVIEW

File: app/assess/templates/sub_criteria.html [hunk 1]
```diff
@@ -44,20 +44,25 @@ <h1 class="govuk-heading-l scoring-heading">{{sub_criteria.id}}</h1>
...
+                    </div>
+                    {% if comments == None %}
+                    <div class="govuk-body govuk-!-margin-top-6">
+                        No comments have been left yet.
+                        </div>
+                    {% else %}
+                    {% for comment in comments %}
+                    <div class="s26-comment-group">
+                      <p class="govuk-body">{{comment.comment}}</p>
+                      <p class="govuk-body-s">{{comment.full_name}}(Assessor) {{comment.email_address}}</p>
```
[the remark this answers is not in the document]
Comment (adamdavies1) [reply app/assess/templates/sub_criteria.html:61]: Yeah was just a place holder, i do need to check when if were passing the role back though the comments endpoint

File: app/assess/views/filters.py [hunk 2]
```diff
@@ -27,7 +27,10 @@ def datetime_format(value, format):
     )
     return formatted_time
 
+def datetime_format_24hr(value):    
+    return datetime.strptime(value, "%Y-%m-%dT%H:%M:%S.%f").strftime("%d/%m/%Y at %H:%M")
```
Comment (stevewalton28) [inline app/assess/views/filters.py:31]: Should consider datetime.fromisoformat(value)?

File: app/static/src/styles/comments.css [hunk 3]
```diff
@@ -0,0 +1,15 @@
+.s26-comment-group {
```
Comment (stevewalton28) [inline app/static/src/styles/comments.css:1]: s26 is a prefix to show the sprint from the prototype - remove all s26 prefixes here

File: app/static/src/styles/comments.css [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+    background-color: #ffffff;
+    padding: 25px 25px 5px;
+    margin-top: 15px;
+    border: 2px #1d70b8 solid;
+}
+
+.s26-comment-group:last-of-type {
```
Comment (stevewalton28) [inline app/static/src/styles/comments.css:8]: s26 is a prefix to show the sprint from the prototype - remove all s26 prefixes here

File: app/static/src/styles/comments.css [hunk 5]
```diff
@@ -0,0 +1,15 @@
...
+    margin-bottom: 30px;
+}
+
+.s26-comment-group .govuk-body-s {
```
Comment (stevewalton28) [inline app/static/src/styles/comments.css:12]: s26 is a prefix to show the sprint from the prototype - remove all s26 prefixes here

**Review by stevewalton28 — CHANGES_REQUESTED** [review]

File: app/static/src/styles/comments.css (hunk 3 above)
Comment (adamdavies1) [reply app/static/src/styles/comments.css]: yup my fault for not copying over from the prototype

**Review by stevewalton28 — APPROVED** [review]

**Pull request merged**